entry.js

```diff
@@ -1,11 +1,9 @@
 import ReactDatepicker from 'react-datepicker'
 import * as CarbonIcons from '@carbon/icons-react'
-import Quill from 'quill'
 
 import * as AdminJSDesignSystem from './src/index'
 
 window.ReactDatepicker = window.ReactDatepicker || ReactDatepicker
 window.CarbonIcons = window.CarbonIcons || CarbonIcons
-window.Quill = Quill
```

Comment:
don't we need the same for tiptap/other editors?

Reply:
I think no, TipTap works without global variables.